### ISSUE TITLE
'keybase' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by sapphirecat** [keybase/keybase-issues#2881]
To import an existing PGP key, the keybase client app is asking me to run "keybase pgp select" but keybase is not on `%PATH%` in Windows. I rebooted, just to make sure.

Normally, I'd dig it out of Program Files or (x86) equivalent, but it's not there. AFAICT. I was also unable to find a CLI inside the app somewhere.

The website tells me it can't be done in the browser, and that I need to use the app. Well, the app simply doesn't work.

**Comment by cjb** [comment]
Your start menu should have a "Keybase Shell" entry that sets it up on the path.

**Comment by sapphirecat** [comment]
Found it. Thanks.

Still, I looked all over the website (especially the [Windows installation](https://keybase.io/docs/the_app/install_windows) and [CLI basics](https://keybase.io/docs/command_line/basics) pages) and the site **always** says "a terminal" and not "Keybase Shell" anywhere. The app's built-in Help just opens the website, so that wasn't very useful either.

Huh. I just discovered that, for some reason, the system tray UI has more menu options than the large, obvious client window ever did. I just didn't notice the tray UI even _had_ a menu bar last night. The white blends in with the browser window that happens to be underneath it. ¯\\\_(ツ)\_/¯

I'll leave this open, you can decide whether any of this warrants any changes to Keybase.  Thanks again.

**Comment by zapu** [comment]
Yeah, it's a bit special on Windows. Visual Studio does similar thing with its "Visual Studio Command Prompt", that's also how some people use `git` with "git shell". So IMO it's consistent for a Windows program.

**Comment by jerome724** [comment]
In case you want it as an global "command" add this path to your path environment ``C:\Users\<windows username>\AppData\Local\Keybase``. Worked fine for me :)
